build.sbt

```diff
@@ -318,10 +318,18 @@ lazy val commonSettings = ReleasePlugin.extraReleaseCommands ++ Seq(
         <url>http://github.com/fwbrasil/</url>
       </developer>
     </developers>)
-) ++ update2_12
+) ++ `scala 2.12 hacks`
 
-lazy val update2_12 = Seq(
-  scalacOptions -= (
-    if (scalaVersion.value.startsWith("2.12.")) "-Xfatal-warnings" else ""
-  )
-)
+lazy val `scala 2.12 hacks` = Seq(
```

Comment:
@fwbrasil Also look here https://github.com/som-snytt/scala/blob/bad61ce0ff9f460c2f8873c134a7f6bee0a53824/src/compiler/scala/tools/nsc/settings/Warnings.scala#L25-L30
For now we use only `imports`